### DISCUSSION TITLE
paretoefficiency

### DISCUSSION THREAD

**Discussion by giscus** [jamesstephenbrown/NZSWebsite#21] [Announcements]
# paretoefficiency

Exploring how Pareto Efficiency can lead to costless improvements in various aspects of life and society.

https://nonzerosum.games/paretoefficiency.html

**Comment by franco3x** [comment]
Another great article. Glad I signed up for the newsletter! I’d never heard of Pareto Efficiency before. Pretty cool. I wonder if it’s hard in practice for mulstiple people to agree that a situation is Pareto Efficient. Ie the scenario where one rich person has all of the resources. You could argue she’d be better off if some of her resources were given away because now less ppl want to rob her lol

**Reply by jamesstephenbrown** [reply]
Thanks Franco, I totally agree - I think you might also argue that there's an optimal amount of wealth required for peak productivity, which means the rich person will benefit from the fruits of higher quality production and innovation, meaning they'll have greater access to better products (and as you say, be able to keep them secure). I think at this stage we're straying into general nuanced non-zero-sum logic, which is great - I also think it's great to have short hand labels like "Pareto Efficient" to refer to something specific, so it can be used as one building-block amongst many in order to understand a situation better. The post on [diminishing returns on happiness](https://nonzerosum.games/diminishingreturns.html) fleshes out some of those mutual-benefit optimisations.

**Comment by andrewtaneglen** [comment]
Would be interesting to add the concept of the future to pareto efficiency. e.g. remaining super-rich in a vastly-unequal world might be pareto-efficient for now, but not necessarily for future generations. Pareto efficiency for people who aim to have children or, more generally, care about the future, might involve being worse off in the near-term, for an eventual future payoff. So the argument 'let's not do anything that makes anyone worse off', should probably be considered in terms of future outcomes as well as the immediate.

**Reply by jamesstephenbrown** [reply]
Good point, I think you're right, but as mentioned above, it's one of those terms where if you think about it enough you can kind of hack it, or find a work-around to make it function to provide actual efficiency, or social well-being or whatever you're after - but I think it's generally useful to have it as a mental shorthand - so you can easily point to an obviously Pareto Efficient situation, and without it having to be a whole solution it is at least a no-brainer improvement.

I was thinking about you when I wrote this actually, realising that this is the sort of logic you were talking about a couple of weeks back about that AI augmented system where you take unnoticeable value from someone and distribute it to where it's most impactful - creating a benefit to the system as a whole without anyone noticing the pinch. I think it was pure coincidence that I was writing about that... but maybe not... who knows?

**Comment by andrewtaneglen** [comment]
I feel the need to question your '...always try to fill it the least amount possible.' example. I'm the opposite, always filling the kettle to the top when boiling it, even refilling and turning it back on in with no clear need for more boiling water in that moment. This is based on the utility of having as much hot water available as possible at all times, on the basis that someone else might also want a hot drink within the time the water remains hot, and that it will take less time to boil in the near future while it retains some warmth.
This does come at the cost of it taking longer to make a single hot drink - so shades of Pareto Inefficiency. BUT, there is future cost of having someone annoyed at you for only boiling a single cup of water, or being annoyed yourself at having to fill and boil again if you want another hot drink soon after.
In this way systems of punishment or justice, or more generally 'future costs' can affect what constitutes Pareto efficiency.  I guess all this comes within the basic sense of 'not worse off', when considering the net outcome of a series of tradeoffs, so doesn't really diverge at all from your point.
